svelte.config.js

```diff
@@ -28,24 +25,28 @@ const config = {
 				}
 			}
 		},
-		adapter: adapter()
+		adapter: adapter({
+			split: true
+		})
 	},
 	preprocess: [
 		mdsvex({
-			extensions: [".svx"],
+			extensions: [".md"],
 
 			smartypants: {
 				dashes: "oldschool"
 			},
 
-			remarkPlugins: [remarkA11yEmoji, remarkSlug, remarkGfm, remarkGithub],
+			remarkPlugins: [remarkA11yEmoji, remarkSlug, remarkGfm, remarkGithub]
 		}),
 		sveltePreprocess({
 			postcss: {
-				plugins: postCssPlugins,
+				plugins: [autoprefixer, cssnano, mediaMinMax]
```

Comment:
Previously, we used `autoprefixer()`, is it correct that we know only use a reference to the function?

Reply:
It seems to work just fine, yes.